style.css

```diff
@@ -1,5 +1,75 @@
+
 ul {
   list-style-type: none;
   padding: 1px;
   margin: 1px;
 }
+
+.black-border {
+  border: 2px solid black;
+}
+
```

Comment:
- [ ] You have this CSS class but it's not being used in your CSS file. The attribute which is `border: 2px solid black;` has however been repeated in your CSS file Kindly let's try and keep the code DRY. One way of doing this is by grouping together all classes using the same attribute eg.

```
input[type=text],
.remove-btn,
#add {
  border: 2px solid black;
}
```
`